File: examples/regex_match_store/regex_match_store.js
const value = `This is target 1.
Name is apple.
Target 2 is here. 
Name is orange.`
if (matches = [...value.matchAll(/target\s([\d+]).*?Name\s+is\s+([^\.]+)\./sig)] ) {
    for (match of matches) {
        console.log(match[1]+"="+match[2]);
    }
}

File: examples/regex_match_store/regex_match_store.pl
my $check_value = $value = "This is target 1.
Name is apple.
Target 2 is here. 
Name is orange.";
$check_value=~ s{target\s([\d+]).*?Name\s+is\s+([^\.]+)\.}{
    print $1.'='.$2."\n";
}gesix;
